scripts/treasury/badger_buyback_ladder_order.py

```diff
@@ -0,0 +1,49 @@
+from decimal import Decimal
+from math import log, floor
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+from great_ape_safe.ape_api.helpers.coingecko import get_cg_price
+
+
+SAFE = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+USDT = SAFE.contract(r.treasury_tokens.USDT)
+BADGER = SAFE.contract(r.treasury_tokens.BADGER)
+
+COW_PROD = True
+
+MAX_USDT = 500_000
+MAX_BADGER = 200_000
+LEG_AMOUNT = 100_000
+DEADLINE = 60 * 60 * 24 * 7  # gives one week to sell
+DECIMALS = USDT.decimals()
+
+
+def main(usdt_remaining=(MAX_USDT * 0.7), badger_bought=64151.1085, approve=True):
+    usdt_remaining_int = int(usdt_remaining)
+    SAFE.init_cow(prod=COW_PROD)
+    if approve:
+        amount_to_approve = (
+            int(usdt_remaining_int / LEG_AMOUNT) * LEG_AMOUNT * 10 ** DECIMALS
```

Comment:
could use a variable to store the calculation result of `int(usdt_remaining_int / LEG_AMOUNT)` since it is used later again. similar for `10 ** DECIMALS`